execution/executor-service/src/main.rs

```diff
@@ -1,26 +1,52 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+use aptos_executor_service::process_executor_service::ProcessExecutorService;
+use aptos_logger::info;
 use clap::Parser;
+use std::net::SocketAddr;
 
 #[derive(Debug, Parser)]
 struct Args {
-    #[clap(long, default_value_t = 8080)]
-    pub server_port: u16,
-
     #[clap(long, default_value_t = 8)]
     pub num_executor_threads: usize,
+
+    #[clap(long)]
+    pub shard_id: usize,
+
+    #[clap(long)]
+    pub num_shards: usize,
+
+    #[clap(long, num_args = 1..)]
+    pub remote_executor_addresses: Vec<SocketAddr>,
+
+    #[clap(long)]
+    pub coordinator_address: SocketAddr,
 }
 
 fn main() {
     // TODO (skedia): Uncomment this once the executor service is implemented.
-    let _args = Args::parse();
+    let args = Args::parse();
     aptos_logger::Logger::new().init();
 
-    // let server_addr = SocketAddr::new(IpAddr::V4(Ipv4Addr::LOCALHOST), args.server_port);
-    // let executor_service =
-    //     ProcessExecutorService::new(server_addr, 1000, args.num_executor_threads);
-    // executor_service.run();
+    let (tx, rx) = crossbeam_channel::unbounded();
+    ctrlc::set_handler(move || {
```

Comment:
why do you need this? I am wondering if we can do some of this clean up when this is dropped..

Reply:
For now this is the main func() that starts the service. How else can we drop this ?

Reply:
Hmm can you move the shutdown function call to `drop` method of `ProcessExecutorService` ?

Reply:
Ok done